config-templates/geo-redundancy-deployment.yaml

```
@@ -29,7 +29,7 @@ server_configs:
   pd:
     replication.location-labels: ["zone","dc","rack","host"]
     replication.max-replicas: 5
-    label-property:
+    label-property: `label-property` is not supported in the TiDB v5.2 and later versions by default. To set the replica policy, use the [placement rules](/configure-placement-rules.md).
```

Please remove the reference format []() because it is in a code block.
```suggestion
    label-property: Since TiDB 5.2, the `label-property` configuration is not supported by default. To set the replica policy, use the [placement rules](/configure-placement-rules.md).
```